be/src/formats/parquet/meta_helper.h

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+#include "exec/hdfs_scanner.h"
+#include "formats/parquet/group_reader.h"
+#include "gen_cpp/Descriptors_types.h"
+#include "metadata.h"
+#include "runtime/descriptors.h"
+
+namespace starrocks::parquet {
+
+class MetaHelper {
+public:
+    virtual ~MetaHelper() = default;
+    virtual void set_existed_column_names(std::unordered_set<std::string>* names) const = 0;
+
+    virtual void build_column_name_2_pos_in_meta(std::unordered_map<std::string, size_t>& column_name_2_pos_in_meta,
+                                                 const tparquet::RowGroup& row_group,
+                                                 const std::vector<SlotDescriptor*>& slots) const = 0;
+
+    virtual const void prepare_read_columns(const std::vector<HdfsScannerContext::ColumnInfo>& materialized_columns,
+                                            std::vector<GroupReaderParam::Column>& read_cols,
```

Comment:
why did you use const void, could you confirm it?

Reply:
I will change it.